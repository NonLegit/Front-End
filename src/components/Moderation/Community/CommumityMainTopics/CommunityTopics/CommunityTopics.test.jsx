import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import CommunityTopics from './CommunityTopics';

// test snapshot
const setTopics = () => {};
test('test snapshot', async () => {
  const props = {
    topics: [],
    setTopics,
  };
  const tree = renderer.create(<CommunityTopics {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
