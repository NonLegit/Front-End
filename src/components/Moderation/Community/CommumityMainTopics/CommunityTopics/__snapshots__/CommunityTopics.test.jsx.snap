// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot 1`] = `
<h2
  className="css-fo6dne"
  onClick={[Function]}
  onTouchEnd={[Function]}
>
  <span
    className="css-1qne1za"
    data-testid="add"
    id="add"
  >
    <div
      className="MuiBox-root css-1290pys"
    />
    <textarea
      className="css-h1mzer"
      data-testid="input"
      id="input"
      onChange={[Function]}
      onFocus={[Function]}
      onInput={[Function]}
      onKeyDown={[Function]}
      type="text"
    />
  </span>
</h2>
`;
